modules/integrations-fragment-collection-contributor/src/main/resources/com/liferay/fragment/collection/contributor/integrations/dependencies/microstrategy/index.js

```diff
@@ -0,0 +1,15 @@
+const baseRestURL = configuration.hostUrl;
+const projectId = configuration.projectId;
+const dossierId = configuration.dossierId;
+const containerHeight = configuration.containerHeight;
+
+const projectUrl = baseRestURL + '/app/' + projectId;
+
+const dossierUrl = projectUrl + '/' + dossierId;
+
+microstrategy.dossier.create({
```

Comment:
Hey @martin-dominguez I just tried testing this, but it's not working for me. Where does `microstrategy` come from? Is this supposed to have the same index.html as the tableau fragment?

Reply:
@ethib137 you're absolutely right... I'm committing a new version of index.html, now it should work for you. 
Apology for the inconvenience